src/router.js

```diff
@@ -242,6 +250,20 @@ export const routes = [
     component: () =>
       import(/* webpackChunkName: "agent" */ '@/pages/Agents/AgentPage.vue')
   },
+  // ---------------------------//
+  //
+  // Integrations
+  //
+  // ----------------------------//
+  {
+    name: 'pagerduty',
+    path: '/:tenant?/pagerduty/:config?',
```

Comment:
Without a config string, this is just a blank page. Might we add a redirect if that config isn't present?